README.md

```diff
@@ -25,22 +22,11 @@ So if you have a directory: `/Users/me/project/protobufs/` that has:
 
 ```sh
 cd ~/my_project/protobufs
-docker run -v `pwd`:/defs namely/protoc-ruby
+docker run -v `pwd`:/defs namely/protoc-all -f myproto.proto -l ruby #or go, csharp, etc
```

Comment:
I've had a hell of a time getting volumes to work like this in bash for windows. `docker-compose`, however, works like a charm.

Reply:
Interesting, with volume mounts? It's probably worthwhile adding a docker-for-windows example too.